pkgs/build-support/rust/rustc-wrapper/rustc-wrapper.sh

```diff
@@ -26,4 +26,5 @@ if (( "${NIX_DEBUG:-0}" >= 1 )); then
     printf "  %q\n" "${extraAfter[@]}" >&2
 fi
 
+export NIX_LDFLAGS="@ldflags@ $NIX_LDFLAGS"
```

Comment:
`NIX_LDFLAGS_@suffixSalt@` is sometimes used for cross compiling, but it's not clear to me if we need it here.

Reply:
Yeah, I can't tell either. Maybe it'll be good to check how CC wrapper works and see if this could affect cross compiled LLVM Rust packages and if it does then add the suffix?

Reply:
@Ericson2314 probably knows. But not sure if his GitHub feed is too swamped as well.